pkg/client/metrics/metrics.go

```diff
@@ -45,8 +45,47 @@ var (
 		},
 		[]string{"path", "server"},
 	)
+
```

Comment:
Can we also have success counters for each of these? So that we can compare how many were successful vs. failed